corehq/sql_db/routers.py

```diff
@@ -98,4 +100,13 @@ def db_for_read_write(model, write=True):
             engine_id = connection_manager.get_load_balanced_read_engine_id(ICDS_UCR_ENGINE_ID)
         return connection_manager.get_django_db_alias(engine_id)
     else:
+        if not write:
+            alias = connection_manager.get_load_balanced_read_engine_id(app_label)
+            cursor = connections[alias].cursor()
+            cursor.execute('SELECT now() - pg_last_xact_replay_timestamp() AS replication_delay;')
```

Comment:
@calellowitz FYI I've asked @sravfeyn to look into making a utility for doing this in a more genarlized way: https://trello.com/c/YD9iWTQU/377-utilities-to-monitor-replication-delay